### PR TITLE
branch length checker v1

### PR DESCRIPTION
**Pull request by brian-l**
```╭─brian.lee@brian-lee ~/git/platform ‹feature/p3-848-sub-approved-filter*›
╰─➤  pre-commit run                                                                                                                                                                                                                       1 ↵
Check JSON...........................................(no files to check)Skipped
Check Yaml...............................................................Passed
Trim Trailing Whitespace.................................................Passed
Check for merge conflicts................................................Passed
Fix End of Files.........................................................Passed
Debug Statements (Python)............................(no files to check)Skipped
Check python ast.....................................(no files to check)Skipped
black................................................(no files to check)Skipped
Neo4J Cypher linter..................................(no files to check)Skipped
GraphQL schema linter................................(no files to check)Skipped
Terraform linter.....................................(no files to check)Skipped
Typescript project linter............................(no files to check)Skipped
Golang file linter...................................(no files to check)Skipped
ts-verify-athena.....................................(no files to check)Skipped
ts-verify-platform-admin.............................(no files to check)Skipped
ts-verify-assessor-web-app...........................(no files to check)Skipped
Branch length checker....................................................Passed
╭─brian.lee@brian-lee ~/git/platform ‹feature/p3-848-sub-approved-filter*›
╰─➤  git checkout -b feature/obtuse-length-max-over-forty-name-for-good-measure
M       .pre-commit-config.yaml
Switched to a new branch 'feature/obtuse-length-max-over-forty-name-for-good-measure'
╭─brian.lee@brian-lee ~/git/platform ‹feature/obtuse-length-max-over-forty-name-for-good-measure*›
╰─➤  pre-commit run
Check JSON...........................................(no files to check)Skipped
Check Yaml...............................................................Passed
Trim Trailing Whitespace.................................................Passed
Check for merge conflicts................................................Passed
Fix End of Files.........................................................Passed
Debug Statements (Python)............................(no files to check)Skipped
Check python ast.....................................(no files to check)Skipped
black................................................(no files to check)Skipped
Neo4J Cypher linter..................................(no files to check)Skipped
GraphQL schema linter................................(no files to check)Skipped
Terraform linter.....................................(no files to check)Skipped
Typescript project linter............................(no files to check)Skipped
Golang file linter...................................(no files to check)Skipped
ts-verify-athena.....................................(no files to check)Skipped
ts-verify-platform-admin.............................(no files to check)Skipped
ts-verify-assessor-web-app...........................(no files to check)Skipped
Branch length checker....................................................Failed
- hook id: branch-length-checker
- exit code: 1

Your branch name should be less than 40 characters to guarantee a namespace deploy
```

### CODE REVIEW

File: branch-length-checker/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# branch-length-checker
+Pre-commit hook that will run `cypher-lint` on `.cypher|.cql|.cpy` files.  See https://github.com/cleishm/libcypher-parser for cypher-lint docs and FAQ.
```
Comment (miriamgrigsby) [inline branch-length-checker/README.md:2]: cypher? wat? 🐃

**Review by miriamgrigsby — CHANGES_REQUESTED** [review]
update those docs!

File: branch-length-checker/README.md (hunk 1 above)
Comment (stephen-bapple) [reply branch-length-checker/README.md:2]: The dark speech of Neo4J

**Review by stephen-bapple — APPROVED** [review]
Aside from comment on docs, LGTM

Epic change BTW

**Review by miriamgrigsby — APPROVED** [review]

**Pull request merged**